test/functional/apps/visualize/index.ts

```diff
@@ -7,115 +7,116 @@
+        './_vertical_bar_chart_nontimeindex',
+        './_line_chart_split_series',
+      ].forEach(load);
     });
 
-    describe('', function () {
+    describe('visualize ciGroup9', function () {
       this.tags('ciGroup9');
 
-      loadTestFile(require.resolve('./_embedding_chart'));
-      loadTestFile(require.resolve('./_area_chart'));
-      loadTestFile(require.resolve('./_data_table'));
-      loadTestFile(require.resolve('./_data_table_nontimeindex'));
-      loadTestFile(require.resolve('./_data_table_notimeindex_filters'));
+      [
+        './_embedding_chart',
+        './_area_chart',
+        './_data_table',
+        './_data_table_nontimeindex',
+        './_data_table_notimeindex_filters',
+      ].forEach(load);
 
       // this check is not needed when the CI doesn't run anymore for the OSS
-      if (!isOss) {
-        loadTestFile(require.resolve('./_chart_types'));
-      }
+      if (!isOss) load('./_chart_types');
     });
 
-    describe('', function () {
+    describe('visualize ciGroup10', function () {
       this.tags('ciGroup10');
 
-      loadTestFile(require.resolve('./_inspector'));
-      loadTestFile(require.resolve('./_experimental_vis'));
-      loadTestFile(require.resolve('./_gauge_chart'));
-      loadTestFile(require.resolve('./_heatmap_chart'));
-      loadTestFile(require.resolve('./input_control_vis'));
-      loadTestFile(require.resolve('./_histogram_request_start'));
-      loadTestFile(require.resolve('./_metric_chart'));
+      [
+        './_inspector',
+        './_experimental_vis',
+        './_gauge_chart',
+        './_heatmap_chart',
+        './input_control_vis',
+        './_histogram_request_start',
+        './_metric_chart',
+      ].forEach(load);
     });
 
-    describe('', function () {
+    describe('visualize ciGroup4', function () {
       this.tags('ciGroup4');
 
-      loadTestFile(require.resolve('./_line_chart_split_series'));
-      loadTestFile(require.resolve('./_line_chart_split_chart'));
-      loadTestFile(require.resolve('./_pie_chart'));
-      loadTestFile(require.resolve('./_point_series_options'));
-      loadTestFile(require.resolve('./_markdown_vis'));
-      loadTestFile(require.resolve('./_shared_item'));
-      loadTestFile(require.resolve('./_lab_mode'));
-      loadTestFile(require.resolve('./_linked_saved_searches'));
-      loadTestFile(require.resolve('./_visualize_listing'));
-      loadTestFile(require.resolve('./_add_to_dashboard.ts'));
-
-      if (isOss) {
-        loadTestFile(require.resolve('./_tile_map'));
-        loadTestFile(require.resolve('./_region_map'));
-      }
+      [
+        './_pie_chart',
+        './_point_series_options',
+        './_markdown_vis',
+        './_shared_item',
+        './_lab_mode',
+        './_linked_saved_searches',
+        './_visualize_listing',
+        './_add_to_dashboard.ts',
+      ].forEach(load);
+
+      if (isOss) ['./_tile_map', './_region_map'].forEach(load);
```

Comment:
This `if (isOss)` will be going away soon in this PR https://github.com/elastic/kibana/pull/94968
Not sure which will get merged first.